include/sbi/riscv_unpriv.h

```diff
@@ -7,37 +7,35 @@
  *   Anup Patel <anup.patel@wdc.com>
  */
 
-#ifndef __SBI_UNPRIV_H__
-#define __SBI_UNPRIV_H__
+#ifndef __RISCV_UNPRIV_H__
+#define __RISCV_UNPRIV_H__
 
 #include <sbi/riscv_encoding.h>
 #include <sbi/sbi_bits.h>
 #include <sbi/sbi_types.h>
 
 #define DECLARE_UNPRIVILEGED_LOAD_FUNCTION(type, insn)			\
-static inline type load_##type(const type *addr, ulong mepc)		\
+static inline type load_##type(const type *addr)			\
 {									\
-	register ulong __mepc asm ("a2") = mepc;			\
-	register ulong __mstatus asm ("a3");				\
+	register ulong __mstatus asm ("a2");				\
 	type val;							\
-	asm ("csrrs %0, "STR(CSR_MSTATUS)", %3\n"					\
+	asm ("csrrs %0, "STR(CSR_MSTATUS)", %3\n"			\
 		#insn " %1, %2\n"					\
-		"csrw "STR(CSR_MSTATUS)", %0"					\
+		"csrw "STR(CSR_MSTATUS)", %0"				\
 	: "+&r" (__mstatus), "=&r" (val)				\
-	: "m" (*addr), "r" (MSTATUS_MPRV), "r" (__mepc));		\
+	: "m" (*addr), "r" (MSTATUS_MPRV));		\
```

Comment:
Can you please restore the alignment of the trailing slash with the slashes on surrounding lines.

Reply:
Thanks for catching. I will fix trailing slash alignment.